.github/ISSUE_TEMPLATE/---intermittent-ui-issue.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "💔 Feature request"
```

Comment:
is this the proper name?

Reply:
No, I did not copied/pasted correctly :(